### PR TITLE
CI: pre install git last verson and git lfs in test

### PR DESCRIPTION
**Pull request by a1012112796**
I have build a docker image base on golang:1.16 and install git v2.30.1 , git lfs. suggest use it instead of installing git lfs before runing test code.

link:
https://hub.docker.com/r/gitea/test_env
Source:
https://gitea.com/gitea/test_env/src/branch/master/Dockerfile

### CODE REVIEW

**Comment by zeripath** [comment]
I think we should put this dockerfile under a Gitea umbrella.

**Comment by lunny** [comment]
@a1012112796 Could you contribute that repository to https://gitea.com/gitea/ ?

**Comment by a1012112796** [comment]
> @a1012112796 Could you contribute that repository to https://gitea.com/gitea/ ?

@lunny  I think it's okay to make this file as a part of gitea, but should have a new repo in docker hub.

File: .drone.yml [hunk 1]
```diff
@@ -964,3 +954,59 @@ steps:
...
+kind: pipeline
+name: test-env-linux-amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+trigger:
+  event:
+  - promote
```
Comment (a1012112796) [inline .drone.yml]: a manually triger. I think it's better use it in here
https://docs.drone.io/promote/

File: contrib/gitea_test_env/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM golang:1.15
+
+RUN echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ buster main contrib non-free" > /etc/apt/sources.list
+RUN echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ buster-updates main contrib non-free" >> /etc/apt/sources.list
+RUN echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ buster-backports main contrib non-free" >> /etc/apt/sources.list
+RUN echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian-security buster/updates main contrib non-free" >> /etc/apt/sources.list
```
Comment (a1012112796) [inline contrib/gitea_test_env/Dockerfile]:
```suggestion
```
If not in China, they are useless.

**Comment by lunny** [comment]
> > @a1012112796 Could you contribute that repository to https://gitea.com/gitea/ ?
> 
> @lunny I think it's okay to make this file as a part of gitea, but should have a new repo in docker hub.

https://gitea.com/gitea/test-env , Please send directly to this repository

**Comment by a1012112796** [comment]
@lunny Done :)
https://gitea.com/gitea/test-env/pulls/1

**Comment by a1012112796** [comment]
review ready :)

File: .drone.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ steps:
 
   - name: lint-backend
     pull: always
-    image: golang:1.16
+    image: gitea/test_env:linux-arm64
```
Comment (gary-kim) [inline .drone.yml]: It'd be nice to have a link to the repo so people know where to look for modifications.
```suggestion
    # https://gitea.com/gitea/test-env
    image: gitea/test_env:linux-arm64
```

**Review by gary-kim — APPROVED** [review]
Would be nice to have the comment on `amd64` images as well but otherwise, LGTM

**Review by lunny — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**